src/db_helper.py

```diff
@@ -60,8 +60,8 @@ def setup_db():
     '  key TEXT UNIQUE NOT NULL,'
     '  author TEXT NOT NULL,'
     '  title TEXT NOT NULL,'
-    '  year INT NOT NULL,'
-    '  booktitle TEXT NOT NULL,'
+    '  year INT,'
+    '  booktitle TEXT,'
```

Comment:
is there some specific reason for this change?

I'm not really sure if these are required or not. On one hand https://www.bibtex.com/t/template-inproceedings/ only includes two required fields, but on the other hand, according to https://www.bibtex.com/f/year-field/ and https://www.bibtex.com/f/booktitle-field/, `year` and `booktitle` are required

Reply:
I made the change as i was following this  https://www.bibtex.com/t/template-inproceedings/  page. I think you can make the shot here on which is used. NOT NULL is way less problematic, but I made the changes for it to work without it and I need to remove some tweaks in this pull request if we choose that.  Namely not null requires this to be changed year=request.form.get('year_inproceedings'), and this to be removed     if info.year == '':
        year_value = None
    else:
        year_value = info.year